src/stores/ToastStore.js

```diff
@@ -43,12 +44,9 @@ export default class ToastStore extends EventEmitter {
 
         const oldIndex = this._toasts.findIndex(t => t.key === newToast.key);
         if (oldIndex === -1) {
-            // we only have two priorities so just push realtime ones onto the front
-            if (newToast.priority) {
-                this._toasts.unshift(newToast);
-            } else {
-                this._toasts.push(newToast);
-            }
+            let newIndex = this._toasts.length;
+            while (newIndex > 0 && this._toasts[newIndex - 1].priority > newToast.priority) --newIndex;
```

Comment:
Among toasts of equal priority, this seems like it keeps the existing behaviour of leaving the newest toast at the bottom of the stack... I guess it's unclear what to do with that case, because I can imagine separate journeys where each possible ordering would be useful.

Seems okay to ignore this for now then.

Reply:
Ah yep - I assumed this was probably the best thing to do: have added some jsdoc to explain the justification.